### PR TITLE
ADR002 for products repo and README update

### PR DESCRIPTION
**Pull request by jemnery**

### CODE REVIEW

File: architecture/decision/records/002-use-products-repo.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# ADR-002 Use a dedicated repository for data product defintions
+
+## Status
+
+🤔 Proposed
```
Comment (bagg3rs) [inline architecture/decision/records/002-use-products-repo.md]: This has been decided now

Comment (jemnery) [reply architecture/decision/records/002-use-products-repo.md]: Fair enough. In that case I'll also add ADR-002 to the index.

File: architecture/decision/records/002-use-products-repo.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- Clean separation of infrastructure code
+- Separation of code validation (for example schema validation)
+- Clear separation of pull requests between platform maintenance and platform usage
+
+## Decision
+
+Consistent with other MOJ platforms, use a dedicated [GitHub repository](https://github.com/ministryofjustice/data-platform-products) to define and manage data products.
+
+## Consequences
+
```
Comment (bagg3rs) [inline architecture/decision/records/002-use-products-repo.md:25]: We will have to manage an additional repository

**Review by julialawrence — APPROVED** [review]
LGTM

**Pull request merged**